lib/msf/core/handler.rb

```diff
@@ -245,11 +245,7 @@ def register_session(session)
 
     # Call the handler's on_session() method
     on_session(session)
-
-    # Process the auto-run scripts for this session
-    if session.respond_to?('process_autoruns')
-      session.process_autoruns(datastore)
-    end
+    session.bootstrap(datastore)
```

Comment:
Thanks guys.

Brent, I saw your other PR and thought "ah, I think I moved/modified this code in the packet pivot PR". So yes you're correct. I'll be sure to merge this once your PR is landed. I'll try to get to that tonight.

Sempervictus: yeah, it's horrid isn't it!